go.mod

```diff
@@ -48,3 +48,5 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+go 1.13
```

Comment:
Is that necessary? I still use 1.12.X since we had a lot of issues with 1.13 and go modules 😄

Reply:
No, I'm going to remove it. It's annoying that go 1.13 does this. :/

Reply:
Also, yes. I'm probably going to revert to 1.12.X.